docs/scriptappy.json

```diff
@@ -4148,6 +4148,10 @@
               "optional": true,
               "defaultValue": "ComponentRefLine~LineLabel",
               "type": "#/definitions/ComponentRefLine/definitions/LineLabel"
+            },
+            "slope": {
```

Comment:
It looks like you haven't re-run spec generation after your last changes of the api, please do, so this get's marked as optional and get the defaultValue: 0